### DISCUSSION TITLE
EVAPORATION OF WATER FROM SPRINKLERS

### DISCUSSION THREAD

**Discussion by MarioCruzM** [firemodels/fds#13364] [General]
Hi, I'm currently working on a sprinkler model, and I wanted to know if there's a way to see (like the smoke) the water vapor created from the evaporation of the water ejected by the sprinklers, I've been reading the user’s guide on "15.3 Liquid Droplets", but I don't get if you can actually see the water vapor. I'm uploading a very simple scenario of the model I'm working with; FYI I’ve been doing the model on pyrosim. Thank you for your help. 
[TEST 4.zip](https://github.com/user-attachments/files/16767598/TEST.4.zip)

**Comment by rmcdermo** [comment]
2D slices are the easiest option.

Or you can use isosurfaces (ISOF) with Smokeview with QUANTITY='MASS FRACTION', SPEC_ID='WATER VAPOR', VALUE=.../

Lastly, you could either use 3D slices and something like pyFDStools or FDSreader from [Third-Party Tools](https://github.com/firemodels/fds/wiki/Third-Party-Tools).

**Reply by MarioCruzM** [reply]
Thank you so much for your help. I did end up using 3D slices for this matter.
